modules/.common

```diff
@@ -9,8 +9,16 @@ conflict payu
 # Name of this module's environment, e.g. payu/VERSION
 set payuname [module-info name]
 
+# Check if modulefile is in prerelease folder
+if {[string match "/g/data/vk83/prerelease/*" "$ModulesCurrentModulefile"]} {
+    set is_prerelease true
+} else {
+    set is_prerelease false
+}
+
```

Comment:
We could just deploy a `.common` modules file into the `prerelease` folder?

Reply:
`env.sh` uses the path to the unpacked conda environment,  `/g/data/vk83/apps/$name/bin/activate` or `/g/data/vk83/prerelease/apps/$name/bin/activate`. So could have a separate `.common` and `env.sh` files with different paths for prerelease apps? In a way having separate files would be good for testing without changing the released payu modulefiles.

Reply:
Changed to deploy a `.common` and `env.sh` to `prerelease` - updated CD to generate files using different paths for release and pre-release files